system.json

```diff
@@ -36,11 +36,6 @@
       "lang": "de",
       "name": "German",
       "path": "lang/de.json"
-    },
-    {
-      "lang": "it",
-      "name": "Italian",
-      "path": "lang/it.json"
```

Comment:
Is this intentional?

Reply:
Foundry printed a warning saying that `lang/it.json` did not exist, so I removed it to avoid that warning. I can add it back, though

Reply:
That's my mistake haha
Should be fixed now

Reply:
Fixed